crates/router/src/connector/globalpay/requests.rs

```diff
@@ -1,13 +1,19 @@
+use common_utils::types::StringMinorUnit;
 use masking::Secret;
 use serde::{Deserialize, Serialize};
 
+#[derive(Debug, Serialize)]
+pub struct GlobalPayRouterData<T> {
+    pub amount: StringMinorUnit,
+    pub router_data: T,
+}
 #[derive(Debug, Serialize)]
 pub struct GlobalpayPaymentsRequest {
```

Comment:
There are more fields in this request which needs amount changes like `cashback_amount`, Please check all fields again